tests/util.c

```diff
@@ -47,6 +47,46 @@ static void _trap_handler(int signal, siginfo_t *info, void *ctx)
 	_exit(161);
 }
 
+/**
+ * Add rules for gcov/lcov
+ * @param ctx the filter context
+ * @param action the action for the rules
+ *
+ * This function is to make it easier for developers to temporarily add support
+ * for gcov/lcov to a test program; it likely should not be used in the normal
+ * regression tests.  Further, this should only be necessary for the "live"
+ * tests.
+ *
+ */
+int util_gcov_rules(const scmp_filter_ctx ctx, int action)
```

Comment:
nitpick - `action` isn't needed.

Reply:
Whops, it should be, nice catch :)  I'll fix that up after you've had a chance to look at the other comments I made (see above); if I force push now I'll likely blow away those commit links ...

Reply:
You're good to go on the force push

Reply:
Thanks.

This *should* be fixed now, but if you notice anything else awry let me know.